pytest/tests/sandbox/patch_state.py

```diff
@@ -2,42 +2,16 @@
 # Patch contract states in a sandbox node
 
 import sys, time
-import base58
 import base64
 import pathlib
 
 sys.path.append(str(pathlib.Path(__file__).resolve().parents[2] / 'lib'))
 
 from cluster import start_cluster
-from configured_logger import logger
 from transaction import sign_deploy_contract_tx, sign_function_call_tx
-from utils import load_test_contract
+from utils import load_test_contract, figure_out_sandbox_binary
```

Comment:
Alternatively change this to `import utils` and use `utils.load_test_contract` and `utils.figure_out_sandbox_binary` in the code.